server.js

```diff
@@ -1,90 +1,32 @@
 'use strict';
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const apiRouter = require('./routes/api.routes');
+const port = process.env.PORT || 3000;
 
-var express = require('express');
-var mongo = require('mongodb');
-var mongoose = require('mongoose');
-var bodyParser = require('body-parser');
-const dns = require('dns');
+// Configurations
+require('./config/db.config');
```

Comment:
Database connection now required in app.js, only 1 line. Your server launch doesn't have to know the logic behind mongoose.connect()